src/Tables/atoms/TypeTableRow.tsx

```diff
@@ -30,6 +30,7 @@ const TypeTableRow : React.FC<IProps> = ({selectable = false, selectCallback, ha
     if(selectCallback){ selectCallback(checked, rowData.id); }
   }, [rowData.id, selectCallback]);
 
+  if(rowData?.columns?.length === 0) { return null;}
```

Comment:
Seems like this is perhaps part of the cause for some strange layout shift moment when data drops in. We can chat about this a bit more on call.

Reply:
Deleted as per conversation